llm-examples/mpt/mpt-7b/Load and inference MPT-7B model.py

```diff
@@ -0,0 +1,74 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # Load and Inference MPT-7B model on Databricks
+# MAGIC
+# MAGIC Environment for this notebook:
+# MAGIC - Runtime: 13.2 GPU ML Runtime
+# MAGIC - Instance: `g5.4xlarge` on AWS
+
+# COMMAND ----------
+
+# MAGIC %pip install xformers einops flash-attn==v1.0.3.post0 triton==2.0.0.dev20221202
+
+# COMMAND ----------
+
+import transformers
+import torch
+
+# COMMAND ----------
+
+name = 'mosaicml/mpt-7b-instruct'
+config = transformers.AutoConfig.from_pretrained(
+  name,
+  trust_remote_code=True
+)
+config.attn_config['attn_impl'] = 'triton'
+config.init_device = 'cuda'
+
+model = transformers.AutoModelForCausalLM.from_pretrained(
+  name,
+  config=config,
+  torch_dtype=torch.bfloat16,
+  trust_remote_code=True,
+  cache_dir="/local_disk0/.cache/huggingface/"
+)
+
+tokenizer = transformers.AutoTokenizer.from_pretrained("EleutherAI/gpt-neox-20b", padding_side="left")
+
+generator = transformers.pipeline("text-generation",
+                                  model=model, 
+                                  config=config, 
+                                  tokenizer=tokenizer,
+                                  torch_dtype=torch.float16,
```

Comment:
Is there a reason this uses float16 rather than bfloat16?

Reply:
No. Let me change it to bfloat16